boards/arm/nrf9160_pca10090/nrf9160_pca10090_common.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	model = "Nordic PCA10090 Dev Kit";
+	compatible = "nordic,pca10090-dk", "nordic,nrf9160-sica",
+		     "nordic,nrf9160";
+
+	chosen {
+		zephyr,console = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 2 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 3 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 4 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 2";
+		};
+		led3: led_3 {
+			gpios = <&gpio0 5 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 3";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 8 GPIO_PUD_PULL_UP>;
+			label = "Switch 1";
+		};
+		button1: button_1 {
+			gpios = <&gpio0 9 GPIO_PUD_PULL_UP>;
+			label = "Switch 2";
+		};
+		button2: button_2 {
+			gpios = <&gpio0 6 GPIO_PUD_PULL_UP>;
+			label = "Push button 1";
+		};
+		button3: button_3 {
+			gpios = <&gpio0 7 GPIO_PUD_PULL_UP>;
+			label = "Push button 2";
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		led3 = &led3;
+		sw0 = &button2;
+		sw1 = &button3;
+		sw2 = &button0;
+		sw3 = &button1;
+	};
+};
+
+&uart0 {
+	status = "ok";
+	current-speed = <115200>;
+	tx-pin = <29>;
+	rx-pin = <28>;
+	rts-pin = <27>;
+	cts-pin = <26>;
+};
+
+&uart1 {
+	status = "ok";
+	current-speed = <115200>;
+	tx-pin = <1>;
+	rx-pin = <0>;
+	rts-pin = <14>;
+	cts-pin = <15>;
+};
+

```

Comment:
Please, add:
```
&adc {
	status ="ok";
};

&i2c0 {
	status = "ok";
	sda-pin = <30>;
	scl-pin = <31>;
};

&spi1 {
	status = "ok";
	sck-pin = <19>;
	mosi-pin = <18>;
	miso-pin = <17>;
};
```

Reply:
> What would happen if a project overlay tried to assign those pins to something else? We have a project where we use them for UART, for example.

You can always reconfigure the pins for this SPI instance (or disable it completely) in the overlay when you need them for a different purpose. This configuration will serve as a starting point only. I wanted to make it similar to what is done for other DKs. But we can use different pins for this SPI. Could you suggest some?

Reply:
I would suppose, @anangl @lemrey we have the same problem with every project that overwrites default PIN configuration, no?

Reply:
@anangl I understand, that shouldn't be a problem then if it can be overridden without errors.

Reply:
Then I will add the proposed pin configuration in the board definition.

Reply:
Done